contexts/citizenship-v1.jsonld

```diff
@@ -19,6 +21,7 @@
 
         "description": "http://schema.org/description",
         "name": "http://schema.org/name",
```

Comment:
These should be updated to match the VCWG v2 context -- once that settles. Hopefully that means just adding an `s` to `https` (i.e., the special language terms having been removed). It looks like we should be doing that with all of the schema.org properties down the line in the context file.